### PR TITLE
apigee/gallery-card-twig: twig created for gallery card section

### PR DESCRIPTION
**Pull request by manishagupta90**

### CODE REVIEW

File: themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  set galleryCard = [
+    'galleryDesc',
+  ]
+%}
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% block content %}
+      <div{{ attributes.addClass(galleryCard) }}>
+        <a href={{ content.field_content_link.0['#url'] }}>{{ content.field_card_title }}</a>
+        {{content.field_description}}
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig]: this should read `{{ content.field_description }}`

File: themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      </div>
+      {{content.field_paragraph_featured_image}}
```
Comment (vaibhavjain-in) [inline themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig]: same as above.

File: themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig (hunk 1 above)
Comment (manishagupta90) [reply themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig]: done

File: themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig (hunk 2 above)
Comment (manishagupta90) [reply themes/custom/srijan_devportal_theme/templates/paragraph/paragraph--gallery-card.html.twig]: done

**Pull request merged**